interface/weno/drugPaidInsert.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * weno drug paid insert
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+
+require_once('../globals.php');
```

Comment:
for these scripts where bring in database stuff, recommend protecting it with an acl to avoid users from running these scripts by placing the address into the web browser.